subsys/bluetooth/controller/ll_sw/ull_iso.c

```diff
@@ -99,9 +99,6 @@ uint8_t ll_setup_iso_path(uint16_t handle, uint8_t path_dir, uint8_t path_id,
 	ARG_UNUSED(codec_config_len);
 	ARG_UNUSED(codec_config);
 
-	isoal_sink_handle_t sink_hdl;
-	isoal_status_t err = 0;
```

Comment:
Suggestion: change here to
```
	isoal_sink_handle_t sink_hdl = 0;
	isoal_status_t err = ISOAL_STATUS_ERR_SINK_ALLOC;
```
and keep rest of the code as is. Initializing `sink_hdl` should technically not be necessary but may help guard against non-ideal compiler heuristics. That change would also resolve the issue with the current code that it reports success even if no path was set up (e.g., if only BIS is enabled).

Reply:
@asmk-ot could you take a look at the proposal above? Thanks!